ui/src/app/machines/views/MachineList/MachineListHeader/ActionFormWrapper/DeployForm/DeployForm.test.js

```diff
@@ -185,6 +185,101 @@ describe("DeployForm", () => {
     ]);
```

Comment:
Might as well make the test `.ts` as well?

Reply:
Done.